app/src/App/routes/index.tsx

```diff
@@ -612,6 +612,18 @@ const accountDetails = customWrapRoute({
     },
 });
 
+const termsAndConditions = customWrapRoute({
+    parent: rootLayout,
+    path: 'terms-and-conditions',
+    component: {
+        render: () => import('#views/TermsAndConditions'),
+        props: {},
+    },
+    context: {
+        title: 'Terms And Conditions',
+        visibility: 'is-authenticated',
```

Comment:
Is this page only accessible to logged-in users?

Reply:
no